src/Faker/Provider/az_AZ/Payment.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Faker\Provider\az_AZ;
+
+class Payment extends \Faker\Provider\Payment
+{
+    /**
+     * @see list of Azerbaijan banks (2016-10-26), source: http://banker.az/banklarin-siyahisi/
```

Comment:
information from this site is copyrighted, please find an open-source source, or remove